prime-router/src/main/kotlin/FakeReport.kt

```diff
@@ -35,41 +35,6 @@ class FakeDataService : Logging {
         context: FakeReport.RowContext,
     ): String {
         val faker = context.faker
-        // creates fake text data
```

Comment:
Note: This wasn't actually deleted, I just moved it to the bottom so I could call other generation functions from within it